src/oop/programs_with_multiple_classes/medium/q8/QuestionText.md

```diff
@@ -0,0 +1,42 @@
+# oop-programs_with_multiple_classes-medium-q8
+
+## Question Information:
+
+- Question Type: Java
+
+## Question Text:
+
+You want to write a program that models objects within a house and different rooms. Make a Java program
+consisting of a `Light` and a `Kitchen`. See the requirements below for each class. Note that parts of each class are
+already provided, you only need to implement the other portions.
+
+## Light
+
+The `Light` class currently has the following items:
+
+- Attributes:
+    - A `boolean` called **isOn** that is not accessible from other classes.
+    - A `String` called **type** that is not accessible from other classes.
+
+You must add the following items to complete the class:
+
+- Constructors:
+    - A constructor which is visible from another class and accepts an `boolean` and a `String` in
+      that order as parameters and initializes the **isOn** and **type** fields with
+      the input values.
+
+## Kitchen
+
+The `Kitchen` class currently has the following items:
+
+- Constructors:
+    - A constructor which is visible from another class
+      and initialize the **size** and **personCapacity** fields.
+    - It should initialize **lights** to an array of **Light** classes with the size of **size** attribute value.
+      You must add the following items to complete the class:
+- Attributes:
```

Comment:
Please rewrite this portion down here. It is not specified which parts to add, and modifications to the constructor should not appear beneath the attributes header.